em++.bat

```diff
@@ -8,4 +8,11 @@
   set EM_PY=python
 )
 
-@"%EM_PY%" "%~dp0\%~n0.py" %*
+@if "%EMCC_CCACHE%"=="" (
+  :: Do regular invocation of em++.py compiler
+  "%EM_PY%" "%~dp0\%~n0.py" %*
+) else (
+  :: Remove the ccache env. var, invoke ccache and re-enter this script to take the above branch.
+  set EMCC_CCACHE=
+  ccache "%~dp0\%~n0.bat" %*
+)
```

Comment:
These launcher scripts are tool-generated (by `tools/create_entry_points.py`), so it want to move forward with this I think we would want to modify that script.

Or as an alternative we would inject the `cache` command around the inner call to `clang` rather than the out call to `python emcc`?

Reply:
Injecting to the inner clang call is considerably weaker in terms of performance than injecting to the outer python emcc call.

Reply:
Updated the create_entry_points.py script.